src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle {
+  shape = 'triangle';
+
+  color: string;
+
+  sides: [number, number, number];
+
+  constructor(color: string, ...sides: [number, number, number]) {
```

Comment:
according to requirements, it should accept sides as separate arguments instead of an array
![image](https://github.com/mate-academy/ts_figures_classes/assets/86923089/9d0efeee-9c09-443b-8754-d20ea95e41ae)